server/remote_cache/digest/digest.go

```diff
@@ -547,8 +547,12 @@ func MissingDigestError(d *repb.Digest) error {
 	}
 }
 
-// String returns the digest formatted as "HASH/SIZE".
+// String returns the digest formatted as "HASH/SIZE" or the string "<nil>"
+// if the digest is nil.
```

Comment:
nit: maybe add a comment about not using this and pointing to digest.Key for using digests as keys?

Reply:
Good call, done